cli/pcluster/commands.py

```diff
@@ -48,20 +46,31 @@
 LOGGER = logging.getLogger(__name__)
 
 
-def _create_bucket_with_batch_resources(stack_name, resources_dir, region):
-    random_string = "".join(random.choice(string.ascii_lowercase + string.digits) for _ in range(16))
-    # bucket names must be at least 3 and no more than 63 characters long
-    s3_bucket_name = "-".join([stack_name.lower()[: 63 - len(random_string) - 1], random_string])
+def create_bucket_with_batch_resources(stack_name, region):
```

Comment:
Minor: why removing the leading underscore?

Reply:
Because now it's included from unit tests.

Reply:
imo this should not determine the scope of the method. If it's a "private" function in our application we should keep it as such and include it with the trailing underscore in the unit tests